scripts/ci.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Invoke xcodebuild
+#
+# \param $1 type of project (Project or Workspace)
+# \param $2 path to  to run against
+# \param $3 scheme to build
+# \param $4 configuration
+# \param $5 sdk
+# \param $6+ build commands
+function invoke_xcodebuild()
+{
+    /usr/bin/xcodebuild \
+        -"$1" "$2" \
+        -scheme "$3" \
+        -configuration "$4" \
+        -sdk "$5" \
+        "${@:6}" \
+        CODE_SIGNING_ALLOWED=NO
```

Comment:
Is this copied from elsewhere? This line might be worth a comment

Reply:
specifically the code signing bit? I basically tried to mimic the xcodebuild invocation from ADO's xcode build task